tests/L0/run_optimizers/test_adam.py

```diff
@@ -232,6 +233,7 @@ def testNative(self):
             
             self.model_.load_state_dict(copy.deepcopy(self.model.state_dict()))
 
+    @largeTensorTest('60GB', 'cuda')
```

Comment:
How about testing on a 2x 40GB configuration where each process had at most 40GB? These tests would be auto-skipped, correct?

Reply:
Yes, it should be current-device only IIUC